nixos/modules/services/networking/unifi.nix

```diff
@@ -3,7 +3,10 @@ with lib;
 let
   cfg = config.services.unifi;
   stateDir = "/var/lib/unifi";
-  cmd = "@${pkgs.jre}/bin/java java -jar ${stateDir}/lib/ace.jar";
+  cmd = ''@${pkgs.jre}/bin/java java \
```

Comment:
I think the common practice in the NixOS codebase looks like this (and is IMO way more readable):

``` nix
{
  # any shiny expressions...
  cmd = ''
    @${pkgs.jre}/bin/java java \
      -jar ...
  '';
}
```